sites/geohub/src/components/controls/RasterLegendContainer.svelte

```diff
@@ -55,19 +40,12 @@
   </div>
 {:else if rasterLegendType === DynamicLayerLegendTypes.INTERVALS}
```

Comment:
probably we need to change this legend type to `simple` and `advanced` only.

see the implementation in `VectorLegendPanel`.